recipes/libevent/all/conanfile.py

```diff
@@ -49,14 +49,10 @@ def configure(self):
 
     def requirements(self):
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1j")
+            self.requires("openssl/1.1.1l")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        # temporary fix due to missing files in dist package for 2.1.11, see upstream bug 863
```

Comment:
What made this workaround obsolete?

Reply:
strip_root, it takes the extracted folder and rename to destination

Reply:
Yes, that helps when the directory names are different, but I don't really understand this inline comment. It is talking about missing files...?

Reply:
Sorry, I though was about strip_root. About missing file is still the same problem, the fix is downloading that specific tar file. If you visit https://github.com/libevent/libevent/releases/tag/release-2.1.12-stable, you will see source code tar file, which is the regular way for CCI, but also there is another tar file named with "release"